organization/app/organization/[id]/view-organization/page.tsx

```
@@ -0,0 +1,559 @@
+      setEditedData((prev) => ({
+        ...prev,
+        facilities: [...(prev.facilities || []), facilityName],
+      }));
+    }
+  };
+
+  const handleLocationChange = (field: keyof ILocation, value: string) => {
+    setEditedData((prev) => ({
+      ...prev,
+      location: {
+        ...prev.location!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)